pkg/controller/controller.go

```diff
@@ -140,6 +149,7 @@ func (c *StashController) RunInformers(stopCh <-chan struct{}) {
 	c.rcQueue.Run(stopCh)
 	c.rsQueue.Run(stopCh)
 	c.jobQueue.Run(stopCh)
+	c.bupcQueue.Run(stopCh)
```

Comment:
Rename `bupcQueue.Run(stopCh)` to `bcQueue.Run(stopCh)`?

Reply:
renamed bupQueue to bcQueue